scheduler/src/cook/kubernetes/api.clj

```diff
@@ -260,7 +262,9 @@
         [(util/make-atom-updater current-nodes-atom) ; Update the set of all pods.
          (util/make-nested-atom-updater pool->node-name->node get-node-pool node->node-name)]
          old-current-nodes @current-nodes-atom]
-    (log/info "In" compute-cluster-name "compute cluster, node watch processing nodes:" (keys @current-nodes-atom))
+    (log/info "In" compute-cluster-name "compute cluster, node watch processing nodes"
```

Comment:
I think we want different log lines. Newlines are seen by us in the log, but doing it as lots of separate log lines avoids creating a single huge unwieldy message.

Reply:
Made the changes we discussed offline.